### DISCUSSION TITLE
Iphone processing - iTunes

### DISCUSSION THREAD

**Discussion by hpcnetworks** [sepinf-inc/IPED#2239] [Q&A]
Hi there!

Is it possible to process iphone data using its itunes backup? I have full access to the phone.

Thanks in advance!

**Comment by lfcnassif** [comment]
In the right way, currently not. It's planned and in the queue: https://github.com/sepinf-inc/IPED/issues/977

You can try to convert the backup before with other tool, like https://github.com/MaxiHuHe04/iTunes-Backup-Explorer

Anyway, iPhone artifacts parsing today is very very limited, improving it is also in the queue: #43

**Comment by hpcnetworks** [comment]
Hi Luis!

Thanks for the quick reply! I just need the WhatsApp parser by now.

With the suggested tool I just need to convert iTunes backup and then
process it with iped? It sounds nice!

Best Regards

Bruno Costa


Em qua., 5 de jun. de 2024 12:56, Luis Filipe Nassif <
***@***.***> escreveu:

**Reply by lfcnassif** [reply]
Specifically for WhatsApp, IPED is able to decode most information processing the unencrypted iTunes backup directly, since IPED doesn't rely on database or attachment names or paths to locate them, most time. But, converting the backup before can be useful for corner cases, like this:
https://github.com/sepinf-inc/IPED/issues/2044
